app/services/resident_list.rb

```diff
@@ -0,0 +1,38 @@
+require 'pdf-reader'
+
+class InvalidFileFormatError < StandardError; end
```

Comment:
This should be within the ResidentList::PDF class. It's going to be namespaced with those names, so you may need to update a test where you reference it. It'll be called `ResidentList::PDF::InvalidFormatError`, which is a little more descriptive if it's ever logged.